src/openvpn/options.c

```diff
@@ -3888,7 +3888,7 @@ check_file_access(const int type, const char *file, const int mode, const char *
     /* Scream if an error is found */
     if (errcode > 0)
     {
-        msg(M_NOPREFIX | M_OPTERR | M_ERRNO, "%s fails with '%s'", opt, file);
+        msg(M_NOPREFIX | M_OPTERR | M_ERRNO, "%s fails with '%s', you can check file Permissions or selinux security context", opt, file);
```

Comment:
How can you be sure that the error is about permissions? It may also be something totally unrelated.

Reply:
a read error or broken device or broken symlink will also trigger this. Also it is missing `"`.